src/wirecloud/platform/workspace/routes.py

```diff
@@ -605,9 +605,9 @@ async def process_mashup(db: DBDep, user: UserDep, request: Request,
     else:
         from_ws = await get_workspace_by_id(db, Id(workspace_id))
         if from_ws is None:
-            return build_error_response(request, 404, "Workspace not found")
+            return build_error_response(request, 404, _("Workspace not found"))
         if not await from_ws.is_accsessible_by(db, user):
-            return build_error_response(request, 403, f"You are not allowed to read from workspace {workspace_id}")
+            return build_error_response(request, 403, _(f"You are not allowed to read from workspace {workspace_id}"))
```

Comment:
No se puede realizar una traducción si ha tenido un reemplazo antes, habría que hacer:
```suggestion
            return build_error_response(request, 403, _(f"You are not allowed to read from workspace %(workspace_id)s") % {"wokspace_id": workspace_id})
```